src/controller/CHIPDeviceController.cpp

```diff
@@ -2316,23 +2316,23 @@ CHIP_ERROR DeviceController::UpdateDevice(NodeId peerNodeId)
 {
     VerifyOrReturnError(mState == State::Initialized && mFabricIndex != kUndefinedFabricIndex, CHIP_ERROR_INCORRECT_STATE);
 
-    OperationalDeviceProxy * proxy = GetDeviceSession(GetPeerScopedId(peerNodeId));
+    OperationalSessionSetup * proxy = GetDeviceSession(GetPeerScopedId(peerNodeId));
     VerifyOrReturnError(proxy != nullptr, CHIP_ERROR_NOT_FOUND);
 
     return proxy->LookupPeerAddress();
 }
 
-OperationalDeviceProxy * DeviceController::GetDeviceSession(const ScopedNodeId & peerId)
+OperationalSessionSetup * DeviceController::GetDeviceSession(const ScopedNodeId & peerId)
 {
-    return mSystemState->CASESessionMgr()->FindExistingSession(peerId);
+    return mSystemState->CASESessionMgr()->FindExistingSessionSetup(peerId);
 }
 
-OperationalDeviceProxy * DeviceCommissioner::GetDeviceSession(const ScopedNodeId & peerId)
+OperationalSessionSetup * DeviceCommissioner::GetDeviceSession(const ScopedNodeId & peerId)
 {
     mSystemState->CASESessionMgr()->FindOrEstablishSession(peerId, &mOnDeviceConnectedCallback,
                                                            &mOnDeviceConnectionFailureCallback);
 
-    // If there is an OperationalDeviceProxy for this peerId now the call to the
+    // If there is an OperationalSessionSetup for this peerId now the call to the
     // superclass will return it.
```

Comment:
This has _very_ different semantics from before.  Before we would return a useful think if the session was already established.  Now we won't.

Reply:
I am 90% sure this will also be removed in https://github.com/project-chip/connectedhomeip/issues/20452